sunpy/timeseries/timeseries_factory.py

```diff
@@ -324,6 +326,11 @@ def _parse_args(self, *args, **kwargs):
                 result = self._read_file(path, **kwargs)
                 data_header_pairs, filepaths = _apply_result(data_header_pairs, filepaths, result)
 
+            # pathlib.Path
+            elif (isinstance(arg, pathlib.Path)):
```

Comment:
It might be better to use `os.PathLike` both here and in Map? (I only learned this existed today).

Reply:
sure, probably better.

Map doesn't use `isinstance` like here, it uses the [_possibly_pathlike] (https://github.com/sunpy/sunpy/blob/master/sunpy/map/map_factory.py#L397)